.github/workflows/pr-build.yaml

```diff
@@ -67,11 +67,15 @@ jobs:
           make yarn_dependencies
       - name: Check format
         run: |
-          yarn prettier --check . || (echo "::error ::Please format your code using 'yarn fmt'"; exit 1)
+          yarn prettier --check . || (echo "::error ::Please format your code by using 'yarn fmt'"; exit 1)
         working-directory: ui
       - name: Build UI
         run: |
           make ui
         env:
           NO_MINIMIZE: true
           CI: true
+      - name: Check type
+        run: |
+          yarn tsc || (echo "::error ::Please check your code type by using 'yarn tsc'"; exit 1)
```

Comment:
Is it possible to integrate tsc + eslint in the normal development flow, just like what we have in the webpack era?

Reply:
done!